Dockerfile.s390x

```diff
@@ -132,7 +132,7 @@ RUN set -x \
 # Install notary and notary-server
 ENV NOTARY_VERSION v0.3.0
 RUN set -x \
-	&& export GO15VENDOREXPERIMENT=1 \
+	&& export GO15VENDOREXPERIMENT=0 \
```

Comment:
should we have a comment as well, explaining why we need to set it to 0 for s390x?